Controller/AdminController.php

```diff
@@ -313,10 +313,16 @@ public function modifyForumAction(Request $request, $id = null)
             $forum = new ForumEntity();   
         }
         
+        $topiccount = ModUtil::apiFunc('ZikulaDizkusModule', 'user', 'countstats', ['id' => $id,
```

Comment:
I will this will be refactored later I'm now focusing on working admin at the moment. I want to keep it working. I would get lost if I would refactor everything at once.